libc-test/build.rs

```diff
@@ -200,6 +213,9 @@ fn main() {
     let target2 = target.clone();
     cfg.field_name(move |struct_, field| {
         match field {
+            "d_namelen" if openbsd && struct_ == "dirent" => "d_namlen".to_string(),
```

Comment:
Could this struct field be updated on openbsd to match the name in C? For renamings like this Rust shouldn't differ much